justfile

```diff
@@ -61,6 +112,36 @@ install-crds: generate
   kubectl create -f crd/restateclusters.yaml
   kubectl create -f crd/restatedeployments.yaml
 
+# Apply CRDs to cluster (create or update)
+apply-crds: generate
+  kubectl apply -f crd/restateclusters.yaml
+  kubectl apply -f crd/restatedeployments.yaml
+
+# Delete CRDs from cluster
+delete-crds:
+  kubectl delete -f crd/restateclusters.yaml --ignore-not-found
+  kubectl delete -f crd/restatedeployments.yaml --ignore-not-found
+
+# Deploy operator to current kubectl context
+deploy: docker
```

Comment:
are you actually using all of these new commands? i am not sure we strictly need them